src/pages/issues/index.tsx

```diff
@@ -10,45 +10,19 @@ import {
 import { ISSUES_URL } from '@/constants/url';
 import { IssueItem } from '@/interfaces/issueItem.type';
 import { PullRequestAndIssueItem } from '@/interfaces/pullRequestIssueItem';
-
-type SearchFieldProps = {
-    onSearchTextSubmitted: (searchString: string) => void;
-    loading: boolean;
-};
-
-const SearchField = ({ onSearchTextSubmitted, loading }: SearchFieldProps) => {
-    const [searchText, setSearchText] = useState<string>('');
-    const onSearchTextChanged = (e: ChangeEvent<HTMLInputElement>) => {
-        setSearchText(e.target.value);
-    };
-
-    return (
-        <form
-            className={classNames.searchFieldContainer}
-            onSubmit={(e) => {
-                e.preventDefault();
-                onSearchTextSubmitted(searchText);
-            }}
-        >
-            <input
-                placeholder="Enter query string to search issues"
-                value={searchText}
-                onChange={onSearchTextChanged}
-                className={classNames.issueSearchInput}
-            />
-            <button
-                className={classNames.issuesSearchSubmitButton}
-                disabled={loading || !searchText.trim()}
-            >
-                Submit
-            </button>
-        </form>
-    );
-};
+import { useRouter } from 'next/router';
+import { getQueryStringFromInput } from '@/utils/getQueryStringFromInput';
```

Comment:
Is this function used inside this component?

Reply:
Removed this import from here.